### DISCUSSION TITLE
最新版报错频繁

### DISCUSSION THREAD

**Discussion by cruzld** [yueeng/moebooru#252] [Q&A]
😂😂放出1.0.7吧我要退回去

**Comment by yueeng** [comment]
v1.0.9也不行吗，把出错信息贴出来我看下。

**Reply by cruzld** [reply]
跳不出报错信息，直接白屏，然后只能清后台重新打开

**Reply by cruzld** [reply]
打开后随便点第一张图片，然后返回点第二张图片必定白屏，无法后退，只能清后台
手机k50pro

**Comment by yueeng** [comment]
试试v1.0.9.1，如果还不行，点击右上角设置->点击关于->然后将机型信息贴出来，我看下。

**Reply by cruzld** [reply]
我怀疑是我手机的问题，关于也是点不开，点了就白屏，其他手机上不会算了认命了，还是谢谢大佬了。

**Comment by yueeng** [comment]
试试这个：[pre-release 202403040719](https://github.com/yueeng/moebooru/releases/tag/202403040719) 看看关于能不能打开？如果还是不行，就用老版本吧： [v1.0.7.1](https://github.com/yueeng/moebooru/releases/tag/v1.0.7.1) 与 [v1.0.7](https://github.com/yueeng/moebooru/releases/tag/v1.0.7).

**Reply by cruzld** [reply]
可以打开关于了，问题也解决了。👍👍👍
===== BuildConfig =====
APPLICATION_ID: "com.github.yueeng.moebooru.yande"
BUILD_TIME: "202403040719"
BUILD_TYPE: "release"
DEBUG: false
FLAVOR: "yande"
VERSION_CODE: 9
VERSION_NAME: "1.0.9.1"

===== Build =====
BOARD: "matisse"
BOOTLOADER: "unknown"
BRAND: "Redmi"
CPU_ABI: "arm64-v8a"
CPU_ABI2: ""
DEVICE: "matisse"
DISPLAY: "TP1A.220624.014"
FINGERPRINT: "Redmi/matisse/matisse:13/TP1A.220624.014/V14.0.10.0.TLKCNXM:user/release-keys"
HARDWARE: "mt6983"
HOST: "pangu-build-component-system-47740-lcmks-qkzd4-s5njg"
HW_TIMEOUT_MULTIPLIER: 1
ID: "TP1A.220624.014"
IS_ARC: false
IS_DEBUGGABLE: false
IS_EMULATOR: false
IS_ENG: false
IS_MIUI: true
IS_TIMINGTRACE: false
IS_TREBLE_ENABLED: true
IS_USER: true
IS_USERDEBUG: false
MANUFACTURER: "Xiaomi"
MODEL: "22011211C"
ODM_SKU: "matisse"
PERMISSIONS_REVIEW_REQUIRED: true
PRODUCT: "matisse"
RADIO: "unknown"
SERIAL: "unknown"
SKU: "unknown"
SOC_MANUFACTURER: "Mediatek"
SOC_MODEL: "MT6983"
SUPPORTED_32_BIT_ABIS: ["armeabi-v7a","armeabi"]
SUPPORTED_64_BIT_ABIS: ["arm64-v8a"]
SUPPORTED_ABIS: ["arm64-v8a","armeabi-v7a","armeabi"]
TAG: "Class java.lang.Class\u003ccom.github.yueeng.moebooru.CrashActivity\u003e cannot access private final static  field java.lang.String android.os.Build.TAG of class java.lang.Class\u003candroid.os.Build\u003e"
TAGS: "release-keys"
TIME: 1676308414000
TYPE: "user"
UNKNOWN: "unknown"
USER: "builder"

===== VERSION =====
ACTIVE_CODENAMES: []
ALL_CODENAMES: "Class java.lang.Class\u003ccom.github.yueeng.moebooru.CrashActivity\u003e cannot access private final static  field java.lang.String[] android.os.Build$VERSION.ALL_CODENAMES of class java.lang.Class\u003candroid.os.Build$VERSION\u003e"
BASE_OS: ""
CODENAME: "REL"
DEVICE_INITIAL_SDK_INT: 31
INCREMENTAL: "V14.0.10.0.TLKCNXM"
KNOWN_CODENAMES: ["HoneycombMr1","HoneycombMr2","Lollipop","Kitkat","Tiramisu","Gingerbread","Cupcake","IceCreamSandwichMr1","JellyBean","IceCreamSandwich","LollipopMr1","M","N","O","P","Q","R","S","Sv2","Base","NMr1","OMr1","JellyBeanMr1","JellyBeanMr2","Donut","Froyo","GingerbreadMr1","EclairMr1","Honeycomb","Eclair01","KitkatWatch","Base11","Eclair"]
MEDIA_PERFORMANCE_CLASS: 31
MIN_SUPPORTED_TARGET_SDK_INT: 23
PREVIEW_SDK_FINGERPRINT: "REL"
PREVIEW_SDK_INT: 0
RELEASE: "13"
RELEASE_OR_CODENAME: "13"
RELEASE_OR_PREVIEW_DISPLAY: "13"
RESOURCES_SDK_INT: 33
SDK: "33"
SDK_INT: 33
SECURITY_PATCH: "2023-02-01"
